zeebe/gateway-grpc/src/main/java/io/camunda/zeebe/gateway/grpc/GrpcErrorMapper.java

```diff
@@ -224,8 +224,6 @@ private Status mapRejectionToStatus(final BrokerRejection rejection) {
         builder.setCode(Code.INTERNAL_VALUE);
         break;
       case UNAUTHORIZED:
-        builder.setCode(Code.UNAUTHENTICATED_VALUE);
-        break;
```

Comment:
❓ Won't this introduce a breaking change since the response code will be different?

Reply:
No, this rejection was added [here](https://github.com/camunda/camunda/pull/26195) as part of the Identity changes. It was never thrown by us before.